### PR TITLE
Jarvis Launch Changes

### PR DESCRIPTION
**Pull request by mmartin4972**
* Specify the -r flag to run only and not build
* Added the export LCM command, which will be executed anytime launch is called
* Added commands to localization launch

### CODE REVIEW

File: jarvis_files/jarvis_cmd/src/jarvis/launch.py [hunk 1]
```diff
@@ -8,70 +8,78 @@
 # Function that parses launch command
 #
 ########################
-def launch_dir(ctx, package, ssh, opts):
+def launch_dir(ctx, package, ssh, only_run, opts):
+    wid = launch_terminal()
+    exec_cmd(wid, "export LCM_DEFAULT_URL='udpm://239.255.76.67:7667?ttl=255'", 
+            ssh=ssh, ip=AUTON_IP, close_terminal = True)
```
Comment (cgiger00) [inline jarvis_files/jarvis_cmd/src/jarvis/launch.py:14]: can you add this for drive too?

**Review by cgiger00 — CHANGES_REQUESTED** [review]

File: jarvis_files/jarvis_cmd/src/jarvis/launch.py (hunk 1 above)
Comment (mmartin4972) [reply jarvis_files/jarvis_cmd/src/jarvis/launch.py:16]: yes

**Comment by mmartin4972** [comment]
Tested on rover

**Review by cgiger00 — APPROVED** [review]
lgtm

**Pull request merged**